pkg/node/node.go

```diff
@@ -114,6 +122,25 @@ func (n Node) CordonAndDrain(nodeName string, reason string) error {
 	if err != nil {
 		return err
 	}
+	// Emit events for all pods that will be evicted
+	if recorder != nil {
+		pods, err := n.fetchAllPods(nodeName)
+		if err == nil {
+			for _, pod := range pods.Items {
+				podRef := &corev1.ObjectReference{
+					Kind:      "Pod",
+					Name:      pod.Name,
+					Namespace: pod.Namespace,
+				}
+				annotations := make(map[string]string)
+				annotations["node"] = nodeName
+				for k, v := range pod.GetLabels() {
+					annotations[k] = v
+				}
+				recorder.AnnotatedEventf(podRef, annotations, "Normal", PodEvictReason, PodEvictMsg)
```

Comment:
Should `"Normal"` also be in a constant, like `PodEvictReason` and `PodEvictMsg`? What are the other possible values for that argument?